web/src/lib/utils/timeline-util.ts

```diff
@@ -217,6 +218,22 @@ export const plainDateTimeCompare = (ascending: boolean, a: TimelinePlainDateTim
   return aDateTime.millisecond - bDateTime.millisecond;
 };
 
+export const plainDateTimeWithFileNameCompare = (
```

Comment:
Under what conditions are the assets sorted client-side? Would it be sufficient to return the data already sorted instead?